.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Two comments here - this file should be deleted, and we need to add a .gitignore that, among other things, ignores the .vs folder.